cloudbuild.yaml

```diff
@@ -0,0 +1,19 @@
+# steps:
+# - name: "gcr.io/cloud-builders/mvn"
+#   args:
+#   - package
+# artifacts:
+#   objects:
+#   // This is using sources from teh Google Cloud Source Repository
+#     location: "gs://${_BUCKET}/"
+#     paths:
+#     - "target/${_TARGET}"
+
```

Comment:
Can you add a comment that explains why you do this in multiple steps? That is, why not:

```
steps:
# Builds the application and deploys it to App Engine using Maven.
- id: 'build_and_deploy'
  name: 'gcr.io/cloud-builders/mvn'
  args: ['package', 'appengine:deploy']
  env:
    - "GOOGLE_CLOUD_PROJECT=sp19-codeu-9-3673"
```

I can understand splitting it into multiple steps if, for example, you did some sort of validation step after the build but before the deployment that could be used to prevent the deployment from happening. But since there isn't such a step, it isn't clear to me why you've broken it down into the separate steps like this.